CalendarApp/BookingRoom/Helpers/SettingsHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingRoom.Helpers
+{
+    public static class SettingsHelper
```

Comment:
Lets be more specific at naming and call it AppSettingsHelper or ApplicationSe..